
2020-05-09 19:32:53
Traceback (most recent call last):
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 46, in <module>
    download(links,data_list)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 17, in internal_wrapper
    return own_function(*args, **kwargs)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 25, in download
    os.chdir(data_dir)                                               ### change to dataset file path
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/jenkins/workspace/datasets/'

2020-05-09 19:33:38
Traceback (most recent call last):
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 46, in <module>
    download(links,data_list)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 17, in internal_wrapper
    return own_function(*args, **kwargs)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 25, in download
    os.chdir(data_dir)                                               ### change to dataset file path
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/jenkins/workspace/datasets/'

2020-05-09 21:23:35
Traceback (most recent call last):
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 46, in <module>
    download(links,data_list)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 17, in internal_wrapper
    return own_function(*args, **kwargs)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 25, in download
    os.chdir(data_dir)                                               ### change to dataset file path
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/jenkins/workspace/datasets/'

2020-05-09 21:27:16
Traceback (most recent call last):
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 46, in <module>
    download(links,data_list)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 17, in internal_wrapper
    return own_function(*args, **kwargs)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 25, in download
    os.chdir(data_dir)                                               ### change to dataset file path
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/jenkins/workspace/datasets/'

2020-05-09 21:37:03
Traceback (most recent call last):
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 46, in <module>
    download(links,data_list)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 17, in internal_wrapper
    return own_function(*args, **kwargs)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 25, in download
    os.chdir(data_dir)                                               ### change to dataset file path
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/jenkins/workspace/datasets/'

2020-05-10 04:26:54
Traceback (most recent call last):
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 46, in <module>
    download(links,data_list)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 17, in internal_wrapper
    return own_function(*args, **kwargs)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 25, in download
    os.chdir(data_dir)                                               ### change to dataset file path
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/jenkins/workspace/datasets/'

2020-05-10 04:28:17
Traceback (most recent call last):
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 46, in <module>
    download(links,data_list)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 17, in internal_wrapper
    return own_function(*args, **kwargs)
  File "/home/Github/Romania_COVID_Analytics/code/web_crawler.py", line 25, in download
    os.chdir(data_dir)                                               ### change to dataset file path
FileNotFoundError: [Errno 2] No such file or directory: '/var/lib/jenkins/workspace/datasets/'
